src/schema.ts

```diff
@@ -1414,7 +1414,7 @@ export const typeDefs = gql`
     programme(path: String, contextId: String): ProgrammePage
     subjects(metadataFilterKey: String, metadataFilterValue: String, filterVisible: Boolean, ids: [String!]): [Subject!]
     topic(id: String!, subjectId: String): Topic
-    topics(contentUri: String, filterVisible: Boolean): [Topic!]
+    topics(contentUri: String!, filterVisible: Boolean): [Topic!]
```

Comment:
Skal det ikke være lov å hente alle topics?

Reply:
Jeg syns egentlig ikke det.
Vi kan lage en egen query eller en eller annen safeguard for det om vi trenger det i fremtiden.

Det er såpass dyrt kall å gjøre ved et uhell, og noe som burde tenkes ganske nøye gjennom dersom vi vil vurdere det :smile: